pom.xml

```diff
@@ -54,7 +54,11 @@
             <artifactId>mockito-core</artifactId>
             <version>4.0.0</version>
             <scope>compile</scope>
-        </dependency>
+        </dependency>  <dependency>
```

Comment:
new lone between </dependency>  and  <dependency>
 <dependency>
---- xyz
</dependency> 
 <dependency>
--- abc
</dependency> 

this is the format